pom.xml

```diff
@@ -827,7 +831,6 @@
               <buildArgs>
                 <buildArg>--no-fallback</buildArg>
                 <buildArg>--no-server</buildArg>
-                <buildArg>--features=com.google.cloud.nativeimage.features.ProtobufMessageFeature</buildArg>
```

Comment:
Nit: update the PR description to explain why this is removed and what `<extensions>true` does?

Reply:
Done.